org.gimp.GIMP.json

```diff
@@ -63,6 +68,12 @@
                 "-Dfonts=false",
                 "-Ddep11=false"
             ],
+            "cleanup": [
+              "/bin",
+              "/include",
+              "/lib/pkgconfig",
```

Comment:
note that the global one handle this.

Reply:
But you said in https://github.com/flathub/org.gimp.GIMP/pull/321#issuecomment-2204525334 that adding this in global would break plug-ins